lib/ansiblelint/formatters/__init__.py

```diff
@@ -41,69 +43,72 @@ def _format_path(self, path):
 
 class Formatter(BaseFormatter):
 
-    def format(self, match, colored=False):
+    def format(self, match: MatchError, colored: bool = False) -> str:
         formatstr = u"{0} {1}\n{2}:{3}\n{4}\n"
         _id = getattr(match.rule, 'id', '000')
         if colored:
-            return formatstr.format(colorize(u"[{0}]".format(_id), Color.error_code),
-                                    colorize(match.message, Color.error_title),
-                                    colorize(self._format_path(match.filename), Color.filename),
-                                    colorize(str(match.linenumber), Color.linenumber),
-                                    colorize(u"{0}".format(match.details), Color.line))
+            return formatstr.format(
+                colorize(u"[{0}]".format(_id), Color.error_code),
+                colorize(match.message, Color.error_title),
+                colorize(self._format_path(match.filename or ""), Color.filename),
+                colorize(str(match.linenumber), Color.linenumber),
+                colorize(u"{0}".format(match.details), Color.line))
         else:
             return formatstr.format(_id,
                                     match.message,
-                                    match.filename,
+                                    match.filename or "",
                                     match.linenumber,
                                     match.details)
 
 
 class QuietFormatter(BaseFormatter):
 
-    def format(self, match, colored=False):
+    def format(self, match: MatchError, colored: bool = False) -> str:
         formatstr = u"{0} {1}:{2}"
         if colored:
-            return formatstr.format(colorize(u"[{0}]".format(match.rule.id), Color.error_code),
-                                    colorize(self._format_path(match.filename), Color.filename),
-                                    colorize(str(match.linenumber), Color.linenumber))
+            return formatstr.format(
+                colorize(u"[{0}]".format(match.rule.id), Color.error_code),
+                colorize(self._format_path(match.filename or ""), Color.filename),
+                colorize(str(match.linenumber), Color.linenumber))
```

Comment:
unrelated?

Reply:
ditto, not trying to slip anything not required.

Reply:
I guess all this formatting change makes it look like a formatting change. You shouldn't force reviewers to force to compare the diff byte-by-byte hoping to find something legit in the middle for the reformatting.